OneCommunityGlobal/HighestGoodNetworkApp
# changhao - unit test for timeentryform

# Description
added new unit tests for TimeEntryForm

## Main changes explained:
resolved some error from the previous version of unit test

## How to test:
1. check into current branch
2. do `npm install` and `npm test TimeEntryForm.test.js` to run this PR locally
3. verify function 7 unit tests are passed

## Screenshots or videos of changes:
<img width="780" alt="屏幕快照 2024-04-04 下午9 18 33" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/104778270/5b8be95a-ce88-4dbe-a13a-bc9b7a5df9dc">
